src/ngraph/codegen/code_writer.hpp

```diff
@@ -30,13 +30,14 @@ namespace ngraph
 class ngraph::codegen::CodeWriter
 {
 public:
-    CodeWriter();
-    std::string get_code() const;
-
-    void operator+=(const std::string&);
-
-    size_t indent;
-
+    CodeWriter()
```

Comment:
I like CodeWriter as header only. Can we move it to somewhere not related to codgen? We should put it in either the ngraph/runtime dir or perhaps ngraph dir. It is used by other backends which don't do codegen so let's move it.

Reply:
We can move it in a separate PR